app/pb_kits/playbook/pb_textarea/docs/_textarea_error.html.erb

```diff
@@ -1 +1,6 @@
-<%= pb_rails("textarea", props: { error: "This field has an error!", label: "Label", rows: 4}) %>
+<%= pb_rails("textarea",
+    props: {
+        error: "This field has an error!",
```

Comment:
@garettarrowood One clarification per our convo. You mentioned error as a prop being a boolean, but here and in the tests it's being passed as a string which is the error message. I fully understand the boolean concept, with that in mind should we start adding a `error_message` prop as well? Or am I missing the pattern?

Reply:
This is a perfect example of why this prop should have a type. Yay!

If UX wants to make the error configurable, then an `error_message` prop sounds good to me. I guess we'd have the option to:

1. Continue to use `error` to conditionally render an error.  Then have a default `error_message` that could be specified by the end user passing that additional prop. OR
2. Only have an `error_message` prop that is indeed a `String`, with the value getting used in the template. And infer that if such a prop is provided that indeed there is an error.

Both of these possibilities sound good to me. I don't know if custom error messages are a thing in other playbook kits.

Reply:
Going with option 1 and a default message of "This field has an error."